modules/tinymce/changelog.txt

```diff
@@ -35,6 +35,7 @@ Version 5.7.0 (TBD)
     Fixed incorrect behaviour when editor is destroyed while loading stylesheets #INT-2282
     Fixed figure elements incorrectly splitting from a valid parent element when editing the image within #TINY-6592
     Fixed inserting multiple rows or columns in a table cloning from the incorrect source row or column #TINY-6906
+    Fixed a bug where `br` elements were not scrolled into view upon insertion #TINY-6964
```

Comment:
```suggestion
    Fixed an issue where new lines were not scrolled into view when pressing Shift+Enter #TINY-6964
```

Reply:
Just changing `a bug` to `an issue` for consistency. Also reworded, as it sounded like if I just inserted any BR anywhere it'd scroll to it now, but it's only applicable to inserting new lines where a BR is added to represent the new line.